guides/source/getting_started.md

```diff
@@ -160,7 +160,7 @@ run the following:
 $ rails --version
 ```
 
-If it says something like "Rails 6.0.0", you are ready to continue.
+If it says something like "Rails Rails 6.1.3.1", you are ready to continue.
```

Comment:
I'm reading this as, could you fix it?

> "Rails Rails 6.1.3.1"

Also, I'm unsure the value in updating this version every time a new release to Rails is made, perhaps a rough rule would be "every minor version" so 6.1.0 would be sufficient here.

Reply:
Oh, good catch! Will change.

Reply:
Fixed, rebased and repushed.